inst/sass-utils/color-contrast.scss

```diff
@@ -74,3 +75,46 @@ $_luminance-list: .0008 .001 .0011 .0013 .0015 .0017 .002 .0022 .0025 .0027 .003
 @function opaque($background, $foreground) {
   @return mix(rgba($foreground, 1), $background, opacity($foreground) * 100);
 }
+
```

Comment:
Add link to the Bootstrap PR that adds some additional context as to why this is here